celerdata/manifest.json

```diff
@@ -0,0 +1,48 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "790d8932-0833-43ac-b9d8-d6d0a4f11517",
+  "app_id": "celerdata",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Gathers CelerData metrics and logs",
```

Comment:
Is your integration also sending in logs to Datadog?

Reply:
Yes, prometheus metrics and logs. In terms of logging, our integration only provides guidance on how to configure the relevant logs in conf.yaml.example.

Reply:
If you are sending in logs, we will need a log pipeline. Dominic will be following up via email on log pipeline documentation and steps